docs/use_cases/fine_grained_authorization/intro.md

```diff
@@ -0,0 +1,180 @@
+# Use Case: Fine-Grained Authorization
+
+This use case demonstrates how to build a high-performance, real-time **policy engine** using Feldera.
+A policy engine enforces access control **rules** that determine which actions users can perform on various
+resources, and under which conditions.
+We focus on a family of access control models called **Fine-Grained-Authorization** (FGA) models, which were
+first popularized by Google's [Zanzibar](https://research.google/pubs/zanzibar-googles-consistent-global-authorization-system/)
+system.
+
+**We demonstrate how Feldera's best-in-class support for incremental computation and mutually recursive SQL queries
+enables users to implement complex authorization policies with only a few lines of SQL, and to
+enforce them efficiently at scale.**
+
+This guide is designed for the following audiences:
+
+* **Application developers seeking to add an efficient authorization layer to their applications.**
+  Example: You’re building a collaborative wiki platform, and need a robust access control layer.
+  By defining a declarative access control policy as a set of rules, you can use Feldera to evaluate these rules across millions of
+  objects and thousands of users cost-effectively.
+
+* **Policy engine developers looking to scale their compute layer.**
+  Example: You’re developing your own FGA engine, similar to [OpenFGA](https://openfga.dev/), and need a query
+  engine that can efficiently evaluate complex rule sets over large dynamic object graphs.
+
+In this guide, we will:
```

Comment:
zanzibar is proud about consistency. you should mention that we offer it as well.